protocols/grpc/hyperstart.proto

```diff
@@ -46,22 +46,25 @@ message StartContainerRequest {
 	string container_id = 1;
 }
 
+message NewProcessResponse {
+	uint32 pid = 1;
+}
+
 message ExecProcessRequest {
 	string container_id = 1;
-	string process_id = 2;
 	StringUser string_user = 3;
 	Process process = 4;
 }
 
 message SignalProcessRequest {
 	string container_id = 1;
-	string process_id = 2;
+	uint32 pid = 2;
```

Comment:
Is it allowed when runtime makes these requests by passing a pid which is not returned from StartContainer() nor ExecProcess(), but the passing pid is actually existing the the container?

Reply:
I prefer keeping "container_id"  here. We can implement a mechanism to signal to "ALL" processes in a container with a SignalProcessRequest{container_id=xxx, process_id=0} 

Something more we can do is "sending a signal 0 means detect whether the process in a container is alive or not". As "man 2 kill" semantics

Reply:
> Is it allowed when runtime makes these requests by passing a pid which is not returned from StartContainer() nor ExecProcess(), but the passing pid is actually existing the the container?

Any scenario?

Reply:
@WeiZhang555 runtime might wrongly pass the incorrect number of pid, or pass an outdated pid when the pid is reused in the guest. The behavior should be documented.

Reply:
@laijs I don't think we should allow for requests with a pid that has not been created by the agent and returned through gRPC. I'll document that.